### PR TITLE
CASMPET-5300-release-notes and CASMTRIAGE-4127

### PR DESCRIPTION
**Pull request by djlapenta**
# Description

Updates to the CSM 1.3 release notes - following sections were changed:
* Monitoring
* Automation Improvements
* Platform Component upgrades
* Security Improvements
* Customer-requested enhancements
* Security Vulnerability Exceptions

# Checklist Before Merging

- [ ] Need better text for security vulnerability exception around cray-uas-mgr
- [ ] Need better text for security vulnerability exception around velero-plugin-for-aws

[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Review by rustydb — APPROVED** [review]
Looks okay, but could use a thorough review from someone more familiar with these release components.

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -90,4 +113,23 @@ See [Deprecated features](introduction/differences.md#deprecated-features).
...
+| `csm-docker/stable/k8s.gcr.io/kube-proxy:v1.21.12`                          | Upstream fixes are needed and are not yet available for the `1.21.12` version of Kubernetes included in CSM 1.3. |
+| `csm-dckr/stable/dckr.io/nfvpe/multus:v3.7`                                 | Upstream fixes are needed and are not yet available, however we have engaged with the project to make a reduced-vulnerability version available.|
+| `csm-docker/stable/docker.io/sonatype/nexus3:3.38.0-1`                      | Upstream fixes are needed to the base image in order to address the remaining vulnerabilities. |
+| `csm-docker/stable/docker.io/velero/velero-plugin-for-aws:v1.3.1`           | New version with some fixes available but missed release content freeze in CSM 1.3, will incorporate in future release. |
+| `csm-docker/stable/velero/velero:v1.7.1`                                    | This image, while in Nexus, is not running on the system.  We are planning on upgrading to it in a future version of CSM. |
+| `csm-docker/stable/cray-uas-mgr:1.21.0`                                     | This will be addressed in a future version of CSM. |
```
Comment (kimjensen-hpe) [inline RELEASE_NOTES.md:135]: We'll also need to add a comment about csm-docker/stable/cray-postgres-db-backup:0.2.3 per this comment https://jira-pro.its.hpecorp.net:8443/browse/CASMPET-5936?focusedCommentId=5018756&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-5018756

**Review by mitchty — APPROVED** [review]

File: RELEASE_NOTES.md (hunk 1 above)
Comment (djlapenta) [reply RELEASE_NOTES.md:134]: Thanks for the reminder, Kim - I added an entry for cray-postgres-db-backup:0.2.3 into the security vulnerability exception list.

Comment (bo-quan) [reply RELEASE_NOTES.md:134]: Looks good, one minor thing: the entry missed a period at the end of the sentence.

**Review by bo-quan — APPROVED** [review]

**Review by jeremy-duckworth — APPROVED** [review]

File: RELEASE_NOTES.md (hunk 1 above)
Comment (djlapenta) [reply RELEASE_NOTES.md:134]: fixed period..

**Pull request merged**